### PR TITLE
Ocamldoc manpage update

### PR DESCRIPTION
**Pull request by ghost**
The PR provides fixes for [bug 7632: the manual page of ocamldoc does not mention -charset option](https://caml.inria.fr/mantis/view.php?id=7632).

### CODE REVIEW

File: man/ocamldoc.m [hunk 1]
```diff
@@ -255,6 +258,9 @@ Remove blank characters until the first asterisk ('*') in each line of comments.
 .I title
 as the title for the generated documentation.
 .TP
+.BR \-text\ \fIfile\fR
```
Comment (Octachron) [inline man/ocamldoc.m]: Would `.BI \-text \ file` not achieve the same resut?

File: man/ocamldoc.m [hunk 2]
```diff
@@ -313,6 +319,10 @@ to the list of directories search for compiled interface files (.cmi files).
 .B \-all\-params
 Display the complete list of parameters for functions and methods.
 .TP
+.BR \-charset\fR\ \fIs\fR
```
Comment (Octachron) [inline man/ocamldoc.m]: Same as above, why not `.BI  \-charset \ s`?

**Review by Octachron — APPROVED** [review]
Aside from potential minor tweaks, this looks good to me.

**Comment by ghost** [comment]
@Octachron 
> Would `.BI \-text \ file` not achieve the same resut?

> Same as above, why not `.BI \-charset \ s`?

You are right. I've updated the PR.

**Comment by xavierleroy** [comment]
Looks good to me.  Merging in trunk and 4.06 branch.

**Comment by xavierleroy** [comment]
Cherry-pick is [4.06 ffaeb34]

**Pull request merged**